tests/test_matrix.py

```diff
@@ -190,32 +190,9 @@ def test_multiplyMatrix():
     pass
 
 
-#################
-# matrix.structure #
-#################
-
-
-def test_transposeMat():
+def test_trace():
 
     mat = getTokens("[1, 2; \
-                    3,  4]")
-    matTranspose = mat.transposeMat()
-    assert matTranspose.__str__() == "[{1.0},{3.0};{2.0},{4.0}]"
-
-    mat = getTokens("[5,8,2;\
-                      12,30,9;\
-                      4,17,7]")
-    matTranspose = mat.transposeMat()
-    assert matTranspose.__str__() == "[{5.0},{12.0},{4.0};{8.0},{30.0},{17.0};{2.0},{9.0},{7.0}]"
-
-
-def test_isSquare():
-
-    mat = getTokens("[1, 2; \
-                      x, z]")
-    assert mat.isSquare()
-
-    mat = getTokens("[1, 2; \
-                      1, 3; \
-                      1, 4]")
-    assert not mat.isSquare()
+                      3, 4]")
+    trace = traceMat(mat)
+    assert trace.__str__() == "[{5.0}]"
```

Comment:
If you have changed `trace` to tokens list instead of **_Matrix_** object you can use `tokensToString()` from **_visma.io.parser_** to assert the check.
```suggestion
    assert tokensToString(mat.traceMat()) == "5.0"
```
Make sure to call `mat.isSquare()` before calling the `traceMat()`.